src/main/java/com/shanebeestudios/skbee/elements/recipe/expressions/ExprDiscoveredRecipes.java

```diff
@@ -0,0 +1,114 @@
+package com.shanebeestudios.skbee.elements.recipe.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.Keyed;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Recipes - Discovered")
+@Description({"Get and modify the discovered recipes of a player",
+        "Requires minecraft 1.13+"})
```

Comment:
```suggestion
@Description("Get and modify the discovered recipes of a player.")
```
I made not of this down further, but since this is a new class, SkBee doesn't support 1.13 anyways.